db/controllers/comments_controllers.js

```diff
@@ -8,11 +9,30 @@ const getCommentsByArticleId = (req, res, next) => {
   ])
 
     .then(([_, result]) => {
-      res.status(200).send({ status: 200, comments: result });
+      res.status(200).send({ comments: result });
     })
     .catch((err) => {
       next(err);
     });
 };
 
-module.exports = { getCommentsByArticleId };
+const postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { body, username } = req.body;
+  if (!body || !username) {
+    res.status(400).send({ msg: "missing argument" });
+  } else
+    return Promise.all([
+      selectArticleById(article_id),
+      selectUserByUsername(username),
```

Comment:
I will completely leave it up to you if you keep these checks in there, but if either the article_id or username doesn't exist then PSQL should throw an error which you could use instead of needing to use seperate check functions.